src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services28.yml

```diff
@@ -34,3 +34,8 @@ services:
         inherit_tags: true
         tags:
             - name: baz
+
+    with_defaults_aliased:
+        alias: 'with_defaults'
```

Comment:
I would remove single quotes here, in order to harmonize it with the [documentation](http://symfony.com/doc/current/service_container/alias_private.html#aliasing). What do you think?